packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/InteractionLayerContainer.js

```diff
@@ -87,6 +77,7 @@ class InteractionLayerContainer extends Component {
             width={width}
           />
         }
+        <PreviousMarks scale={scale} />
```

Comment:
Yes, this is intentional because when someone makes new marks from previous transcriptions from Caesar and goes to the next question step, if it isn't rendered on top, they no longer can see that they made a new mark and see the mark from caesar on top instead.